src/index.js

```diff
@@ -88,13 +89,19 @@ export default function ({ Plugin, types: t }) {
   }
 
   /**
-   * Infers a displayName from either a class node, or a createClass() call node.
+   * Infers a displayName from either a class node, createClass() call, or a stateless component variable declaration
    */
-  function findDisplayName(node) {
+  function findDisplayName(node, parent) {
     if (node.id) {
       return node.id.name;
     }
+
     if (!node.arguments) {
+      // likely stateless function component. Check if assigned to a variable
```

Comment:
threading the `parent` node through to here so that the following example resolves a displayName of `MyComponent`

```
const MyComponent = (props) => <div />
```